tools/webpack/script-modules.js

```diff
@@ -15,33 +15,37 @@ const { baseConfig, plugins } = require( './shared' );
 
 module.exports = {
 	...baseConfig,
-	name: 'interactivity',
+	name: 'script-modules',
 	entry: {
-		index: './packages/interactivity',
-		debug: './packages/interactivity/src/debug',
-		router: './packages/interactivity-router',
-		navigation: './packages/block-library/src/navigation/view.js',
-		query: './packages/block-library/src/query/view.js',
-		image: './packages/block-library/src/image/view.js',
-		file: './packages/block-library/src/file/view.js',
-		search: './packages/block-library/src/search/view.js',
+		'interactivity/index': './packages/interactivity',
+		'interactivity/debug': './packages/interactivity/src/debug',
+		'interactivity/router': './packages/interactivity-router',
+
+		'block-library/file': './packages/block-library/src/file/view.js',
```

Comment:
Sharing my notes: `glob` for view files would help here with maintenance. Feel free to ignore them as I need to collect some ideas next to the code I see.